projects/packages/jetpack-mu-wpcom/src/features/media/heif-support.php

```diff
@@ -51,14 +79,35 @@ function jetpack_wpcom_maybe_convert_heif_to_jpg( $filename ) {
 	return true;
 }
 
+/**
+ * Log information about HEIF/HEIC conversions.
+ *
+ * @param array $params The parameters to log.
+ *
+ * @return void
+ */
+function jetpack_wpcom_maybe_log_heif_to_jpg( $params = array() ) {
```

Comment:
It looks like wpcomsh has some code that logs to logstash (see `logstash_log_global_styles`). Can we take a similar approach here, and log in both contexts?